src/main/java/org/spongepowered/api/service/economy/EconomyService.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.service.economy;
+
+import java.util.Collection;
+import java.util.Set;
+import java.util.UUID;
+
+import org.spongepowered.api.service.context.Context;
+import org.spongepowered.api.service.context.ContextualService;
+
+import com.google.common.base.Optional;
+
+/**
+ * TODO: overview doc
+ *
+ */
+public interface EconomyService extends ContextualService {
+
+	/**
+	 * Retrieves the default {@link Currency} used by the {@link EconomyService}.
+	 * 
+	 * @return {@link Currency} default for the EconomyService
+	 * 
+	 * @see Currency
+	 */
+	Currency getDefaultCurrency();
+
+	/**
+	 * Returns the {@link Collection} of supported {@link Currency} objects that are
+	 * implemented by this EconomyService.
+	 * 
+	 * @return Collection of all Currencies
+	 */
+	Collection<Currency> getCurrencies();
+	
+	Optional<Account> getAccount(String identifier);
+
+	Collection<Account> getAccounts(UUID uuid);
```

Comment:
What exactly is the use of this method?

There are player-bound accounts (Wallet) and unbound accounts (String ID).

Maybe it would make sense to replace the String id with an UUID.